app/src/androidTest/java/net/skyscanner/backpack/button/BpkButtonTest.kt

```diff
@@ -50,104 +50,120 @@ class BpkButtonTest(flavour: Flavor) : BpkSnapshotTest() {
     )
 
   @Test
-  fun text() = capture {
+  fun text() {
```

Comment:
this makes sure we're not starting an activity for every variant, even if we don't run them